index.js

```diff
@@ -57,6 +57,22 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "roll",
+    (msg) => {
+        try {
+            rollDecider(stripContent(msg.content));
```

Comment:
As it stands, it seems like you are calling `rollDecider` once, throwing it away, and then calling it again and returning it. Why not return the first one?